instructions/COORDINATOR.md

```diff
@@ -0,0 +1,21 @@
+
+# Coordinate the ceremony
+
+## Prerequisites
+
+You will need Rust to compile the code that verifies the contributions. You can follow the recommended [guide](https://www.rust-lang.org/tools/install).
+
+## Coordinate the participants
+
+First, verify the [contributions](https://github.com/dusk-network/powersoftau-attestations) of Zcash up to number 87 using this [guide](VERIFY.md). This will output a challenge file you need to send to the first participant on the list.
```

Comment:
~~This~~ **The last step of this verification** will output a challenge file you need to send to the first participant on the list.